attiny/attiny.ino

```diff
@@ -17,6 +17,7 @@
 #include <avr/io.h>
 #include <avr/wdt.h>
 #include <avr/sleep.h>
+#include <RunningAverage.h>
```

Comment:
Where is this coming from? Does this library need to checked in as well?

Reply:
I just had them in my global libraries but will add them.

Reply:
Thanks. It makes it easier for someone to get started if everything is there.

I would have been fine with you documenting the required dependencies and how to install them too.